### DISCUSSION TITLE
Should we programm as we explain human the ideas?

### DISCUSSION THREAD

**Discussion by manuelgarciaortega** [official-stockfish/Stockfish#3895] [Ideas]
1. If there are only pawn in the column a or h
2. If the side that has the pawn/s has only bishop/s of different colour to the coronation square.
3. If the weak side has his king controlling the coronation square.

It doesn't matter how many bishops with different colour to the coronation square, pawns in the column a or h it is draw as it is impossible to bring out the king.

https://github.com/official-stockfish/Stockfish/issues/3894

**Comment by vondele** [comment]
with NNUE such hand-coded evaluation terms are not added any longer, the network will learn this if there are sufficient positions like this in the training set.
